src/main/java/me/robifoxx/block/Main.java

```diff
@@ -168,24 +168,9 @@ public void onEnable() {
                                 if(found) {
                                     if(!f_type.equalsIgnoreCase("DISABLED")) {
                                         loc.getWorld().spawnParticle(Particle.valueOf(f_type), loc.getX(), loc.getY(), loc.getZ(), f_quan, f_dx, f_dy, f_dz, f_speed);
-//                                        ParticleEffect.valueOf(f_type).display(
-//                                                f_dx,
-//                                                f_dy,
-//                                                f_dz,
-//                                                f_speed,
-//                                                f_quan,
-//                                                loc, pl);
-                                    }
```

Comment:
This causes syntax error, either add it back or use `if(cond) expr; ` syntax

Reply:
Thanks, fixed.